CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # 7.0.0, in progress
 
+## Added
+* Veneur can be set to flush steady OK values for service checks only once per configurable time interval. Veneur will still emit non-OK values and changes to OK from other values. Thanks, [antifuchs][antifuchs]!
+
+[antifuchs]: https://github.com/antifuchs
```

Comment:
What is this voodoo?

Reply:
The other kind of markdown links! I got tired of spelling out the github link every time ((: